lib/Core/BuildEngine.cpp

```diff
@@ -965,6 +965,8 @@ class BuildEngineImpl {
     
     // Find the cycle by searching from the entry node.
     struct WorkItem {
+      WorkItem(Rule * node) { this->node = node; }
+
```

Comment:
/cc @DougGregor sorry to bug you but is it not legal C++14 to initialize this type with `WorkItem{ node }`? I'm trying to understand why Clang and MSVC aren't consistent in this case. I remember there is some subtly w.r.t. single item ctors and initializer_list but I've forgotten the specifics...